control_plane/src/background_process.rs

```diff
@@ -209,7 +209,14 @@ pub fn stop_process(immediate: bool, process_name: &str, pid_file: &Path) -> any
 }
 
 fn fill_rust_env_vars(cmd: &mut Command) -> &mut Command {
-    let mut filled_cmd = cmd.env_clear().env("RUST_BACKTRACE", "1");
+    // enable RUST_BACKTRACE if it's not configured already, but pass through
+    // RUST_BACKTRACE=full|0 or whatever
```

Comment:
It took me a while to understand the comment. Suggestion:
```suggestion
    // If RUST_BACKTRACE is set, pass it through. But if it's not set, default
    // to RUST_BACKTRACE=1.
```